jordan_bundy/lib/greet.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+var exports = module.exports = {};
+
+var greet = function greet(name) {
```

Comment:
You want either a function expression like:
`var greet = function() {};`
or a declaration like:
`function greet() {};`
not both